appcelerator/titanium_mobile
# fix(ios): animating view width/height from zero

**JIRA:**
https://jira.appcelerator.org/browse/TIMOB-27236

**Summary:**
- On iOS, animating a view's width/height doesn't work if current width or height is zero. The animation "complete" event will never fire either.
- Not an issue on Android.

**Test:**
1. Build and run `AnimationSizeTest.js` attached to [TIMOB-27236](https://jira.appcelerator.org/browse/TIMOB-27236).
2. Notice the window is white.
3. Tap on the "Expand/Collapse" button.
4. Verify an orange view expands from top/left corner and fills the window. _(This is the fix.)_
5. Tap on the "Expand/Collapse" button again.
6. Verify the orange view collapses to the top/left corner and disappears.
